tools/vsimporter/src/vswriter/VSTemplateParameters.cpp

```diff
@@ -64,6 +65,16 @@ void VSTemplateParameters::setValue(const std::string& key, const std::string& v
     m_params[key] = value;
 }
 
+void VSTemplateParameters::setExtensions(const std::vector<std::string>& extensions) {
+    if (extensions.empty()) {
+        m_params["$extensions$"] = "";
+    } else {
+        std::string extensionsOutput = std::accumulate(extensions.cbegin(), extensions.cend(), std::string("<Extensions>\n"));
+        extensionsOutput += "  </Extensions>";
+        m_params["$extensions$"] = extensionsOutput;
+    }
+}
```

Comment:
You don't seem to be using this function anymore. Revert changes in VSTemplateParameters.cpp/h